fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/AppBarTokens.kt

```diff
@@ -28,9 +28,17 @@ enum class AppBarSize {
     Small
 }
 
```

Comment:
same here, let's not restrict with these 4 possibilities

Reply:
Updated